lib/galaxy/tools/imp_exp/imp_history_from_archive.xml

```diff
@@ -1,7 +1,7 @@
 <tool id="__IMPORT_HISTORY__" name="Import History" version="0.1" tool_type="import_history">
   <type class="ImportHistoryTool" module="galaxy.tools"/>
   <action module="galaxy.tools.actions.history_imp_exp" class="ImportHistoryToolAction"/>
-  <command interpreter="python">unpack_tar_gz_archive.py "${ str( $__ARCHIVE_SOURCE__ ).encode( 'base64' ) }" "${ str( $__DEST_DIR__ ).encode( 'base64' ) }" --$__ARCHIVE_TYPE__ --encoded</command>
+  <command>python $__tool_directory__/unpack_tar_gz_archive.py '${ str( $__ARCHIVE_SOURCE__ ).encode( 'base64' ) }' '${ str( $__DEST_DIR__ ).encode( 'base64' ) }' --$__ARCHIVE_TYPE__ --encoded</command>
```

Comment:
`'$__tool_directory__/unpack_tar_gz_archive.py'`

Reply:
I added quotes. Thanks @bgruening.

Reply:
For the record, I don't think this is necessary and we don't do this in the tool XSD (https://docs.galaxyproject.org/en/master/dev/schema.html#id2).

Reply:
I'm wrong, definitely necessary.